flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -39,30 +39,19 @@ under the License.
 
 	<packaging>jar</packaging>
 
-	<dependencyManagement>
-		<dependencies>
-			<!-- Common dependency of calcite-core and flink-* -->
-			<dependency>
-				<groupId>com.google.guava</groupId>
-				<artifactId>guava</artifactId>
-				<version>19.0</version>
-			</dependency>
-			<!-- Common dependency of calcite-core and flink-table-runtime-blink -->
-			<dependency>
-				<groupId>org.codehaus.janino</groupId>
-				<artifactId>commons-compiler</artifactId>
-				<version>${janino.version}</version>
-			</dependency>
-			<!-- Common dependency of calcite-core and flink-table-runtime-blink -->
-			<dependency>
-				<groupId>org.codehaus.janino</groupId>
-				<artifactId>janino</artifactId>
-				<version>${janino.version}</version>
-			</dependency>
-		</dependencies>
-	</dependencyManagement>
-
 	<dependencies>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.codehaus.janino</groupId>
+			<artifactId>commons-compiler</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.codehaus.janino</groupId>
+			<artifactId>janino</artifactId>
+		</dependency>
```

Comment:
Remove them?? The orignal code is `dependencyManagement` which is different from `dependencies`. 

Btw, Janino is already depended.

Reply:
The `dependencyManagement` should be specified in parent pom to control the dependency versions. The old `dependencyManagement` definitions doesn't really work, i believe it is a mis-use.

`Janino is already depended`, that is the reason why i exclude it from calcite-core because we have a dependency explicitly.